external/mmdetection/detection_tasks/apis/detection/inference_task.py

```diff
@@ -243,8 +246,10 @@ def infer(self, dataset: DatasetEntity, inference_parameters: Optional[Inference
             self.confidence_threshold = self._hyperparams.postprocessing.confidence_threshold
 
         update_progress_callback = default_progress_callback
+        return_saliency_map = False
```

Comment:
this method does not return saliency maps even if it is True. Maybe better name?

Reply:
Fixed